examples/user_guide/Deploy_and_Export.ipynb

```diff
@@ -22,9 +22,24 @@
     "\n",
     "    jupyter labextension install @pyviz/jupyterlab_pyviz\n",
     "    \n",
+    "Note that to use certain components such as Vega, LaTeX and Plotly plots in a notebook the models must be loaded using the extension, e.g.:\n",
+    "\n",
+    "    pn.extension('vega', 'katex')\n",
+    "    \n",
+    "will ensure that the Vega and LaTeX JS dependencies are loaded. Additionally any external ``css_files``, ``js_files`` and ``raw_css`` should be declared in the extension. The ``js_files`` should be declared as a dictionary mapping from the exported JS module name to the URL containing the JS components, while the ``css_files`` can be defined as a list:\n",
```

Comment:
I know this is already merged, but while it's still fresh in your mind, it would be good to clarify the situations where one would need external CSS or JS files.